package.json

```diff
@@ -40,29 +40,24 @@
     "@types/jest": "^24.0.14",
     "@types/react": "^16.8.20",
     "@types/react-dom": "^16.8.4",
-    "@typescript-eslint/eslint-plugin": "^1.10.2",
-    "@typescript-eslint/parser": "^1.10.2",
     "css-loader": "^3.0.0",
     "docz": "^1.2.0",
     "docz-theme-default": "^1.0.0",
-    "eslint": "^5.16.0",
-    "eslint-config-prettier": "^4.3.0",
-    "eslint-plugin-import": "^2.17.3",
+    "eslint": "^6.3.0",
```

Comment:
I'm assuming all of those things that got removed are now a part of this new `eslint` package?